sql/core/src/test/scala/org/apache/spark/sql/execution/PlannerSuite.scala

```diff
@@ -621,6 +621,25 @@ class PlannerSuite extends SharedSQLContext {
       requiredOrdering = Seq(orderingA, orderingB),
       shouldHaveSort = true)
   }
+
+  test("SPARK-24242: RangeExec should have correct output ordering") {
+    val df = spark.range(10).orderBy("id")
```

Comment:
why do we put an `orderBy` in the query?

Reply:
I used it to check Sort elimination. Removed now.